### ISSUE TITLE
Explore phase checklist

### ISSUE THREAD

**Issue by grange85** [nationalarchives/ds-digital-downloads#1]
**READ BEFORE STARTING EXPLORE**
(NOTE: explore = discovery in the service manual)

[GOV.UK Service Manual: How the discovery phase works](https://www.gov.uk/service-manual/agile-delivery/how-the-discovery-phase-works)

---

## Explore DSS checklist
Owned by Delivery Managers @ruthroberts @julieguthrie 

- [ ] Agree that feature/service meets relevant elements of the [Digital Service Standard](https://www.gov.uk/service-manual/service-standard)

## Explore service assessment
Owned by Delivery Managers @ruthroberts @julieguthrie 

- [ ] Agree whether service assessment needed
- [ ] Arrange date
- [ ] Github project board up-to-date, wiki up-to-date etc.
- [ ] Team available to answer questions on the day

## Explore communications checklist
Owned by Product Owners & Delivery Managers @sdwilkes @mcieslak @ACTNA @ruthroberts @julieguthrie 

- [ ] roadmap distributed across business
- [ ] establish stakeholders
- [ ] where relevant, approval process agreed and buy-in received from stakeholders
- [ ] establish who we need for the alpha phase
- [ ] create plan, timeline, availability of team, meetings etc.
- [ ] if relevant discuss what to name proposed service
- [ ] decide whether the feature is a go or no go into alpha
- [ ] where relevant, get approval for go/no go from appropriate channel e.g. discovery board, exec team, UAG, DESG

## Explore UX checklist
Owned by UX team @jklepfer @bethmorr 

### STAGE ONE
- [ ] Desk research/competitor review
All of these steps start in Explore then are revisited in Alpha, Beta and Live
- [ ] Define Users/User group - TEAM
- [ ] Define the business case (and ensure work aligns with this) - PRODUCT OWNER/DELIVERY MANAGER
- [ ] Define hypothesis/product/service - TEAM
- [ ] Define User Goals - TEAM
- [ ] Define Current user journey (and potential user journey) incl pain points - UX and TEAM

### STAGE TWO
In all phases, Explore to Live
- [ ] Ensure research and design are inclusive (WCAG 2.1 guidelines) TEAM (UX led)
- [ ] Design research plan (methods, tasks etc ) - UX
- [ ] Consider any laws or policies that may apply (incl GDPR) TEAM
- [ ] Clarify and agree any objectives for user testing - TEAM
- [ ] Define metrics for testing - UX and TEAM
- [ ] Consider assisted digital - TEAM

### STAGE THREE
All phases Explore to Live
- [ ] Organise and plan practical elements of user research (recruitment/dates/equipment etc)- UX and TEAM
- [ ] Carry out User Testing UX (with team if available)

### STAGE FOUR
All phases Explore to Live
- [ ] Transcription of Data from research - UX
- [ ] Analysis of data - UX and ANALYTICS

### STAGE FIVE
All phases Explore to Live
- [ ] Prioritise User needs - TEAM (UX LED)
- [ ] Distribute/Discuss research findings - TEAM (UX LED)
- [ ] Document User Insights and create design artefacts such as user journeys, storyboards etc - UX

**ITERATE and REFINE!**

## Explore business need checklist

Owned by Delivery Managers & Product Owners @sdwilkes @mcieslak @ruthroberts @julieguthrie @vcorker 

- [ ] Policies, laws etc. to consider, with constraints etc.
- [ ] The policy that relates to your service and how it might prevent you from delivering a good service to your users
- [ ] Business needs/requirements gathered from finance, legal, data protection, IT security, CEE, marketing, risk owner etc.
- [ ] Non-functional requirements, impact on existing technology, dependencies, load & performance, penetration testing, work on mobile, accessibility, all browsers, on site/off site
- [ ] Create a document/card that details:
  * user, business market evidence, analysis and insights
  * dependencies (people, projects, business)
  * what's in/out of scope
  * objectives, definition of done, metrics & analytics
  * prioritised list of user needs and user stories
  * decide the features your ‘minimum viable product’ must have
  * issues and risks

## Explore metrics checklist
Owned by @hoake 

- [ ] What do we currently know about usage and behaviours [Metrics and åUX, Team]. What are they looking at, how did they get there, what did they interact with, did they see the feature? This may involve a number of different qualitive/quantative metrics, including but not exclusive to GA, surveys, UX research, business reports (refunds, digital downloads).
- [ ] Are there any gaps in our knowledge (this will add to what we need to track). What didn't we find out? [TEAM]
- [ ] Tracking outcomes based on user stories and business goals. Why are we doing this? Include UX and business goals. [TEAM]
- [ ] Identify key metrics for success [TEAM]

## Explore technology checklist
Owned by front end & back end dev @mcieslak @ACTNA 

- [ ] Review needs and requirements and discuss
- how you'd start developing a new service if your explore finds there’s a user need for one
- how to meet government accessibility requirements
- how you might build a technical solution given the constraints any legacy systems
- on-going maintenance/support required
- issues and risks
- initial t-shirt estimate
- [ ] Create a document/github wiki that details the above
